plugins/clock/src/Clock.cc

```diff
@@ -68,7 +68,20 @@ namespace gazebo
 
     void GazeboYarpClock::Load(int _argc, char **_argv)
     {
-        m_network = new yarp::os::Network();
+        // To avoid deadlock during initialation if YARP_CLOCK is set,
```

Comment:
Suggested change:
initialation -> initialization

Reply:
Thanks, feel free to do this suggestions with GitHub's integrated system!